.github/workflows/tests.yml

```diff
@@ -0,0 +1,49 @@
+name: Tests
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
```

Comment:
For tests we probably want to build in Debug mode with sanitizers etc. right?

Reply:
Sure, and I've just checked, that the tests still work afterwards with Debug.